src/pivot-table/components/helpers/get-item-size-handler.ts

```diff
@@ -33,7 +33,7 @@ export const getColumnWidthHandler =
   ({ list, listValues, isLastRow, getRightGridColumnWidth }: ColumnWidthHandlerProps): ItemSizeHandler =>
   (colIndex: number) => {
     const cell = isLastRow ? list[colIndex] : listValues[colIndex];
-    // const measureInfoCount = layoutService.layout.qHyperCube.qMeasureInfo.length;
+    // const measureInfoCount = layoutService.visibleMeasureInfo.length;
```

Comment:
maybe remove this line? (or do we keep it there for future reference?)

Reply:
I will keep it for now. Would need to investigate if that that will be needed down the road.